### ISSUE TITLE
[JS][Accessibility] [Toggle Visibility is not setting aria-expanded property to true/false on change of visibility]

### ISSUE THREAD

**Issue by niravk-microsoft** [microsoft/AdaptiveCards#5075]
# Platform

What platform is your issue or question related to? (Delete other platforms).

- [ ] JavaScript

# Author or host

We are using JS Renderer to render adaptive cards and we are also sending adaptive cards json to Outlook. Also plan is to send Adaptive cards to teams as well

If you're an author, who are you sending cards to?
Outlook and future plan is to send to teams

# Version of SDK

AdaptiveCards 2.0.5

# Details

We have noticed that Toggle Visibility is not setting aria-expanded property to true/false on change of visibility

**Comment by RebeccaAnne** [comment]
Toggle Visibility as designed does not allow us to deduce expanded/collapsed for individual elements. We would need to either enhance toggle visibility or design a new mechanism to handle this scenario accessibly.

**Comment by niravk-microsoft** [comment]
What I meant was like using toggle visibility, we are expanding section and displaying further section or we can hide that section too. However reader is not reading and not giving enough explanation from accessibility perspective that specific section is visible or hide. So one way is like if we set aria-expanded as true and false based on visibility of Toggle Visibility, read reads that this section is expanded and collapsed accordingly.

**Comment by jonmill** [comment]
@niravk-microsoft - would ShowCard work for you in the interim?

**Comment by paulcam206** [comment]
Closing due to lack of response.

Addressing this issue is not currently on our roadmap. We may return to this in the future. Feel free to continue the conversation in this issue.
